invenio_mail/ext.py

```diff
@@ -67,5 +67,11 @@ def init_config(self, app):
     def print_email(self, message, app):
         """Print mail to stream."""
         with self._lock:
-            self.stream.write("{0}\n{1}\n".format(message.as_string(), '-'*79))
-            self.stream.flush()
+            invenio_mail = app.extensions['invenio-mail']
+            invenio_mail.stream.write(
+                "{0}\n{1}\n".format(message.as_string(), '-'*79))
+            invenio_mail.stream.flush()
+
+    def send_email(self, message):
```

Comment:
IMHO this should not be here. Users of `Invenio-Mail` should decide how they are going to send the email.